src/php/inc/custom-post-types/recipe.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Is this file supposed to be checked in?

Reply:
Ope, good catch, that must've snuck in while I was testing things out. Removed in https://github.com/sparkbox/sparkpress-wordpress-starter/pull/124/commits/7b89a84908a9625bce8c404d42c5d7e46b84618f